mcp-servers/snow/src/snow/server.py

```diff
@@ -24,6 +24,13 @@
 logger = logging.getLogger(SERVICE_NAME)
 auto_tracing_run(SERVICE_NAME, logger)
 
+# Configure logging for MCP server
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+)
+logger = logging.getLogger(__name__)
```

Comment:
Great to have it! At this point I think we can pass this logger to the `auto_tracing_run(SERVICE_NAME, logger)`. Simplifying the collaboration. I don't know what you think about it.

Reply:
But still needs some more fixup, will do that.